src/keystore/index.ts

```diff
@@ -1,2 +1,8 @@
+import * as config from './config'
 export * from './config'
-export * from './basic'
+
+
+export const getKeyByName = async (keyName: string): Promise<string> => {
```

Comment:
Is this the same as `crypto.keystore.exportSymmKey`?

Reply:
ah yup :+1: I meant to delete this. good looks